### PR TITLE
Remove mutating/defaulter webhooks

### PR DESCRIPTION
**Pull request by koct9i**

### CODE REVIEW

File: api/v1/ytsaurus_webhook.go [hunk 1]
```diff
@@ -45,36 +47,6 @@ func (r *Ytsaurus) SetupWebhookWithManager(mgr ctrl.Manager) error {
...
-							consts.YTComponentLabelName: fmt.Sprintf("%s-%s", r.Name, "yt-master"),
-						},
-					},
-					TopologyKey: "kubernetes.io/hostname",
-				},
-			},
-		}
-	}
-}
-
```
Comment (l0kix2) [inline api/v1/ytsaurus_webhook.go:77]: Do this affinities not needed or set somewhere else?

**Review by l0kix2 — COMMENTED** [review]
Will defaults be applied at all without hook?

File: api/v1/ytsaurus_webhook.go (hunk 1 above)
Comment (koct9i) [reply api/v1/ytsaurus_webhook.go:77]: Not always needed. I've added validation instead.

**Comment by koct9i** [comment]
> Will defaults be applied at all without hook?

Yep. Defaults should be applied on read automatically.

https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting

**Review by l0kix2 — APPROVED** [review]

**Pull request merged**